example/javalib/linting/2-checkstyle/src/InputWhitespaceCharacters.java

```diff
@@ -12,8 +12,10 @@ final class InputWhitespaceCharacters {
    * @return hack
    * @throws java.lang.Exception abc
    **/
-  int test1(int badFormat1, int badFormat2,
-  		final int badFormat3) // violation 'Line contains a tab character.'
+  int test1(
+      int badFormat1,
+      int badFormat2,
+      final int badFormat3) // violation 'Line contains a tab character.'
```

Comment:
TODO: Need to restore this lint error and find a way to allow it as an exception. (Can't finish today.)

Reply:
Done